dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ExecutorServiceImpl.java

```diff
@@ -187,11 +183,20 @@ public Map<String, Object> execProcessInstance(User loginUser, long projectCode,
             return result;
         }
 
+        if(!checkScheduleTimeNum(commandType,cronTime)){
+            putMsg(result, Status.SCHEDULE_TIME_NUMBER);
+            return result;
+        }
+
+        if(!checkScheduleTimeRepeat(commandType,cronTime)){
```

Comment:
I think we should delete duplicate data here instead of return error. This will improve the user experience.

Reply:
ok